src/app/shared/components/mentee-list-popup/mentee-list-popup.component.ts

```diff
@@ -48,6 +49,10 @@ export class MenteeListPopupComponent implements OnInit {
     this.enrolledMenteeList = data;
   }
 
+  closePopup(){
+    this.modalController.dismiss();
+  }
+
   // onPaginatorChange(data: any) {
```

Comment:
remove if not needed

Reply:
pagination is required in 3.0 so will keep this